src/Bicep.Core.UnitTests/Configuration/ConfigurationManagerTests.cs

```diff
@@ -101,7 +101,8 @@ public void GetBuiltInConfiguration_NoParameter_ReturnsBuiltInConfigurationWithA
     ""sourceMapping"": null,
     ""paramsFiles"": null,
     ""userDefinedTypes"": null,
-    ""userDefinedFunctions"": null
+    ""userDefinedFunctions"": null,
```

Comment:
minor: Does this need real experimental feature names, or can it be fake ones so we don't have to chanbge every time?

Reply:
@shenglol, you wrote the test. What are your thoughts on this?

Reply:
resolving as not an issue per offline conversation with @StephenWeatherford